README.asciidoc

```diff
@@ -212,19 +212,21 @@ NOTE: You can use the link:https://github.com/mikery/vagrant-ami[Vagrant-AMI] pl
 
 Install plugin from rubygems:
 ----
-$ vagrant plugin install vagrant-openstack-plugin
+$ NOKOGIRI_USE_SYSTEM_LIBRARIES=1 vagrant plugin install vagrant-openstack-plugin
```

Comment:
I don't believe so. For a few people including myself it was problematic to compile nokogiri. Especially on mac. First it needs too much `-devel` packages and IIRC at the time I couldn't make it compile although I tried to track everything needed. In my opinion it's also recommended to use dynamic system libraries to receive timely security updates.